sufia.gemspec

```diff
@@ -49,6 +49,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'flipflop', '~> 2.2'
   spec.add_dependency 'jquery-datatables-rails', '~> 3.4.0'
   spec.add_dependency 'rdf-rdfxml'
+  spec.add_dependency 'sprockets-rails', '~> 3.1.1'
```

Comment:
Here's the diff of the bundle install https://gist.github.com/jcoyne/4ca52ed0e6bc5f95e666717ce35db4a5